mono/utils/atomic.c

```diff
@@ -582,5 +582,9 @@ InterlockedCompareExchange64(volatile gint64 *dest, gint64 exch, gint64 comp)
 }
 
 #endif
+#endif
 
+#if defined(HOST_WIN32) && defined(_MSC_VER)
```

Comment:
We try to keep glib.h as close to the original as possible, no a great place for this sort of thing.